src/mranderson/log.clj

```diff
@@ -0,0 +1,36 @@
+(ns ^:no-doc mranderson.log
+  "Here we replicate lein logging.
+  This gives us what a lein plugin user would expect and is also reasonable for non-lein use.
+  We don't use's lein's logging methods because we don't want to tie ourselves to lein just for logging.
+  We also add in multi-threading support which lein logging does not do.")
+
+;; lein-isms for controlling logging
+(def ^:private log-info+? (not (System/getenv "LEIN_SILENT")))
```

Comment:
```suggestion
(def ^:private log-info+? (and (not (System/getenv "LEIN_SILENT"))
                               (not (System/getenv "SILENT"))))
```

In case some consumer happens to be picky about using an env var named `LEIN*` :)

Reply:
Thanks @vemv! Not a bad idea, but I'm good with the way it is now.
We only look at LEIN_SILENT for lein compatibility and don't expect anybody to ever use it. The README will probably document DEBUG but not LEIN_SILENT.